debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/SourceInfoTest.java

```diff
@@ -150,6 +152,44 @@ public void shouldRecoverSourceInfoFromOffsetWithNonZeroBinlogCoordinatesAndNonZ
     }
 
     @Test
+    public void shouldRecoverSourceInfoFromOffsetWithFilterData() {
+        final String databaseWhitelist = "a,b";
+        final String tableWhitelist = "c.foo,d.bar,d.baz";
+        Map<String, String> offset = offset(10, 10);
+        offset.put(SourceInfo.DATABASE_WHITELIST_KEY, databaseWhitelist);
+        offset.put(SourceInfo.TABLE_WHITELIST_KEY, tableWhitelist);
+
+        sourceWith(offset);
+        assertThat(source.hasFilterInfo());
```

Comment:
`assertThat()` just returns an assertion object which requires the actual assertion method to be invoked on:

    assertThat(source.hasFilterInfo()).isTrue();

The same a bit below:

    assertThat(source.getDatabaseBlacklist()).isNull();

Reply:
fixed